Raywire/iReporter
# [Finishes #163001610] Improves user experience for incident owner and non-owner

**What does this PR do?**
Hides functionality to edit and delete an incident for a non-owner
Converts date to a user-friendly format
**Descriptions of the tasks to be completed?**
Only the owner of an incident should be able to access the edit and delete functionality.
The date in UTC timezone should be converted to local time zone
The creation date of an incident is now in a human-readable format
**How should this be manually tested?**
Go to https://raywire.github.io/iReporter/UI/interventions.html or https://raywire.github.io/iReporter/UI/redflags.html 
Notice the created on time format
![screenshot from 2019-01-05 11-38-51](https://user-images.githubusercontent.com/8179091/50722331-80ba7c00-10de-11e9-891d-a3298b81d6a2.png)

Navigate to any incident that belongs to a different user and notice that edit and delete functionality are now not accessible
**Pivotal tracker story**
#163001610
![screenshot from 2019-01-05 11-37-35](https://user-images.githubusercontent.com/8179091/50722326-5bc60900-10de-11e9-9f7f-067b29533498.png)
